src/foam/nanos/controller/resetPassword.html

```diff
@@ -0,0 +1,26 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <script language="javascript" src="../../../../../../foam2/src/foam.js"></script>
+    <script language="javascript" src="../../../../../../foam2/src/foam/nanos/nanos.js"></script>
+    <script language="javascript" src="../../../../../../foam2/src/foam/u2/ListCreateController.js"></script>
+    
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500" rel="stylesheet">
+  </head>
+  <body>
+    <!-- Instantiate FOAM Application Controller -->
+    <!-- App Color Scheme, Logo, & Web App Name -->
+    <foam 
+      class="foam.nanos.controller.ResetPasswordController" 
```

Comment:
I think the class name is incorrect here. It should be foam.nanos.auth.forgotPassword.ResetPasswordController